src/main/java/de/rwth/idsg/steve/web/controller/Ocpp16Controller.java

```diff
@@ -53,8 +53,8 @@
  * @author Sevket Goekay <sevketgokay@gmail.com>
  * @since 15.03.2018
  */
-@Controller
-@RequestMapping(value = "/manager/operations/v1.6")
+//@Controller
+//@RequestMapping(value = "/manager/operations/v1.6")
 public class Ocpp16Controller extends Ocpp15Controller {
```

Comment:
Something like that should work

```suggestion
*/
@Controller
@RequestMapping(value = "/manager/operations/v1.6")
@ConditionalOnExpression("'${ocpp.security.profile}' == '0'")
public class Ocpp16Controller extends Ocpp15Controller {
```

Reply:
i am in favor of supporting the superset of functionality and showing it as is, even though that particular installation might not use it or not be configured for it. it does not hurt.

moreover, if we start with conditionals like this, should we also discuss about the new `EVENTS AND CERTIFICATES` menu? should we conditionally disable it as well? or maybe some of its menu items? maybe `INSTALLED CERTIFICATES` and `SIGNED CERTIFICATES` only? where do we stop? therefore: lets support and show all.

i could have put the new operations/functions into the existing `Ocpp16Controller`, but i opted against it for better code separation/organization. so, this new `Ocpp16ImprovedSecurityController` exists for better code compartmentalization only actually. and it was not meant for activating/deactiving functionality conditionally.